core-compose/src/androidTest/java/com/squareup/workflow/ui/compose/ComposeViewFactoryTest.kt

```diff
@@ -47,18 +48,18 @@ class ComposeViewFactoryTest {
         }
 
     composeRule.setContent {
-      // This is valid Compose code, but the IDE doesn't know that yet so it will show an
-      // unsuppressable error.
-      RootView(viewEnvironment = viewEnvironment)
+      AndroidView(RootView(ContextAmbient.current).apply { setViewEnvironment(viewEnvironment) })
```

Comment:
Why doesn't the view need to be remembered here, like in the previous commit?

Reply:
It actually never _needs_ to be remembered, but that keeps the view from being unnecessarily recreated every time we rerender the composition. 

That's unnecessary here in the UI test, _but_, full disclosure, I did the `remember` _after_ I had already changed this test. For consistency, I can change it, but it doesn't really matter, I don't think.